### ISSUE TITLE
Suns appear/disappear while on screen

### ISSUE THREAD

**Issue by royfalk** [vegastrike/Vega-Strike-Engine-Source#564]
Suns are drawn and not drawn when on the screen.
This issue is best seen in a system other than cephid_17.

I think what is happening here is that the sun itself is small and what we see is a halo. If the sun is obscured by something, the halo  (which isn't obscured) isn't drawn.

**Comment by stephengtuggy** [comment]
Yes, I have seen this phenomenon. I'm pretty sure you're right about what's happening.
